service.subtitles.thaisubtitle/service.py

```diff
@@ -0,0 +1,368 @@
+# -*- coding: utf-8 -*-
+#from pydev import pydevd
+#pydevd.settrace('localhost', port=5555, stdoutToServer=True, stderrToServer=True,
+#                suspend=True,
+#                trace_only_current_thread=False
+#)
+
+
+import os
+import sys
+import urllib2
+import urlparse
+import xbmc
+import xbmcvfs
+import xbmcaddon
+import xbmcgui
+import xbmcplugin
+HAS_XBMC = True
+import urllib
+import shutil
+import unicodedata
+import re
+import string
+import difflib
+import HTMLParser
+from thaisubtitles import getallsubs, search_manual, search_movie
+
+__addon__ = xbmcaddon.Addon()
```

Comment:
In your next update please remove \__name\__ style labels, these are for python internals.